packages/utilities/src/utilities.client.ts

```diff
@@ -158,18 +158,25 @@ export function normalizeUrl(url: string, keepFragment?: boolean) {
 }
 
 // Helper function for markdown rendered marked
-// Renders links outside apify.com in readme with rel="noopener noreferrer nofollow" and target="_blank" attributes
-export function markedSetNofollowLinks(href: string, title: string, text: string) {
+// If passed hostname, it renders links outside that hostname in readme with rel="noopener noreferrer" and target="_blank" attributes
+// And links outside apify.com in readme with rel="noopener noreferrer nofollow" and target="_blank" attributes
+export function markedSetNofollowLinks(href: string, title: string, text: string, hostname?: string) {
```

Comment:
So the new `hostname` param is the hostname of the current page (the one we're rendering the link on), right? Maybe sth like `referrerHostname` (or even passing the entire URL, not just the hostname) would help with readability a bit?